### PR TITLE
Actions

### PR DESCRIPTION
**Pull request by WezSieTato**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,13 +16,11 @@ fastlane add_plugin moronepo
 
 Run dart pub moronepo with fastlane!
 
-**Note to author:** Add a more detailed description about this plugin here. If your plugin contains multiple actions, make sure to mention them here.
-
+Moronepo is a tool to simplify development in a dart multi-package repository.
```
Comment (arturdm) [inline README.md]: You can link to the `moronepo` repository here.

File: spec/moronepo_activate_action_spec.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+describe Fastlane::Actions::MoronepoActivateAction do
+  describe 'moronepo_activate' do
+    it 'should run proper shell command' do
```
Comment (arturdm) [inline spec/moronepo_activate_action_spec.rb:3]: How does it handle errors? Is it straightforward for the user what went wrong?

File: spec/moronepo_run_action_spec.rb [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+        .to receive(:sh)
+        .with("flutter pub global run moronepo run -- flutter packages get")
+
+      ActionRunner.moronepo_run("arguments: 'flutter packages get'")
+    end
+
+    it 'should run shell command with working_directory param' do
+      expect(Fastlane::Actions)
+        .to receive(:sh)
+        .with("flutter pub global run moronepo --working-directory nepo_dir run -- flutter packages get")
```
Comment (arturdm) [inline spec/moronepo_run_action_spec.rb:14]: `nepo_dir`? :)

**Review by arturdm — CHANGES_REQUESTED** [review]

File: spec/moronepo_activate_action_spec.rb (hunk 2 above)
Comment (WezSieTato) [reply spec/moronepo_activate_action_spec.rb:3]: It print error to console and fail lane:
```
[11:57:02]: --------------------------
[11:57:02]: --- Step: moronepo_run ---
[11:57:02]: --------------------------
[11:57:02]: $ flutter pub global run moronepo --working-directory example_projects --filter hasTests run -- flutter test
[11:57:03]: ▸ Running "flutter test" for project_with_tests project
[11:57:03]: ▸ Project directory "example_projects/project_with_tests"
[11:57:05]: ▸ 00:01 +0 -1: loading /Users/marcin.s/Documents/Meniga/fastlane-plugin-moronepo/example_projects/project_with_tests/test/i_am_a_test.dart [E]
[11:57:05]: ▸ Failed to load "/Users/marcin.s/Documents/Meniga/fastlane-plugin-moronepo/example_projects/project_with_tests/test/i_am_a_test.dart":
[11:57:05]: ▸ Compilation failed
[11:57:05]: ▸ Test: /Users/marcin.s/Documents/Meniga/fastlane-plugin-moronepo/example_projects/project_with_tests/test/i_am_a_test.dart
[11:57:05]: ▸ Shell: /Users/marcin.s/flutter/bin/cache/artifacts/engine/darwin-x64/flutter_tester
[11:57:05]: ▸ 00:01 +0 -1: Some tests failed.
[11:57:05]: ▸ Compiler message:
[11:57:05]: ▸ Failed to load test harness. Are you missing a dependency on flutter_test?
[11:57:05]: ▸ pub finished with exit code 1
```

File: spec/moronepo_run_action_spec.rb (hunk 3 above)
Comment (WezSieTato) [reply spec/moronepo_run_action_spec.rb:14]: It was first name that i have in mind. The second was `moron_dir`

**Review by arturdm — APPROVED** [review]

**Pull request merged**